server/routes/mongoRoutes.js

```diff
@@ -9,25 +9,25 @@ router.use(express.urlencoded());
 router.post('/', (req, res) => {
   db.createItem(req.body.images)
     .then((rows) => res.status(200).json(rows))
-    .catch((err) => res.status(400).json({ message: `Error: ${err}` }));
+    .catch((err) => res.status(500).json(err));
```

Comment:
If you want to prevent internal information accidentally going out to the public via your error messages, you could place custom text here instead.

Reply:
Good idea! Will do so before merging.